docs/content/ingestion/update-existing-data.md

```diff
@@ -56,13 +67,14 @@ For example
 
 #### `multi`
 
-It is a composing inputSpec to combine two other input specs. It is useful for doing delta ingestion. Note that this is not idempotent operation, we might add some features in future to make it idempotent.
+It is a composing inputSpec to combine two other input specs. It is useful for doing delta ingestion. 
+Note that this is not idempotent operation, we might add some features in future to make it idempotent.
```

Comment:
"not an idempotent operation".  
Then I'd start  new sentence "We might add .."